clang/lib/Format/UnwrappedLineFormatter.cpp

```diff
@@ -57,7 +57,10 @@ class LevelIndentTracker {
   /// Update the indent state given that \p Line is going to be formatted
   /// next.
   void nextLine(const AnnotatedLine &Line) {
-    Offset = getIndentOffset(*Line.First);
+    const auto *Tok = Line.First;
+    if (Tok->is(TT_AttributeMacro) && Tok->Next)
```

Comment:
Shouldn't this be a loop?

Reply:
I thought of it but decided to not bother because it would be rare and looping through all `AttributeMacro` prefixes would still leave comments and function-like macros unsupported. See a better solution in #95634.